test/integration/patients/patient/patient-flow.js

```diff
@@ -429,142 +424,116 @@ context('patient flow page', function() {
+              form: getRelationship(testForm),
+            },
+          }),
+        ];
 
-        fx.included.push({
-          id: '3',
-          type: 'program-actions',
+        const testProgramFlow = getProgramFlow({
           attributes: {
-            name: 'Should not show - unpublished',
-            published_at: null,
-            archived_at: null,
-            behavior: 'standard',
-            details: '',
-            days_until_due: 1,
-            sequence: 2,
+            name: 'Program Flow',
           },
           relationships: {
-            owner: { data: { id: '11111', type: 'teams' } },
-            form: { data: { id: '11111' } },
+            'program-actions': getRelationship(testProgramActions, 'program-actions'),
           },
         });
 
-        fx.included.push({
-          id: '4',
-          type: 'program-actions',
-          attributes: {
-            name: 'Should not show - archived',
-            published_at: testTs(),
-            archived_at: testTs(),
-            behavior: 'standard',
-            details: '',
-            days_until_due: 1,
-            sequence: 3,
-          },
+        fx.data = mergeJsonApi(testFlow, {
           relationships: {
-            owner: { data: { id: '11111', type: 'teams' } },
-            form: { data: { id: '11111' } },
+            'program-flow': getRelationship(testProgramFlow),
           },
         });
 
-        fx.included.push({
-          id: '5',
-          type: 'program-actions',
-          attributes: {
-            name: 'Should not show - automated behavior',
-            published_at: testTs(),
-            archived_at: null,
-            behavior: 'automated',
-            details: '',
-            days_until_due: 1,
-            sequence: 4,
-          },
-          relationships: {
-            owner: { data: { id: '11111', type: 'teams' } },
-            form: { data: { id: '11111' } },
-          },
+        _.each(testProgramActions, programAction => {
+          fx.included.push(programAction);
         });
 
-        fx.included.push({
-          id: '1',
-          type: 'program-flows',
-          attributes: {
-            name: 'Program Flow',
-          },
-          relationships: {
-            'program-actions': {
-              data: [
-                {
-                  id: '1',
-                  type: 'program-actions',
-                },
-                {
-                  id: '2',
-                  type: 'program-actions',
-                },
-                {
-                  id: '3',
-                  type: 'program-actions',
-                },
-                {
-                  id: '4',
-                  type: 'program-actions',
-                },
-                {
-                  id: '5',
-                  type: 'program-actions',
-                },
-              ],
-            },
-          },
-        });
+        fx.included.push(testProgramFlow);
```

Comment:
I just realized this is pushing program data onto the include of getFlow.. which isn't where this data is coming from.. we should be able to construct the test data above the `.routeFoos` and then use them for relationships in this route but still add them to where they should be coming from..

Reply:
Still working on this.

I'm not tracking where the data is supposed to be coming from?

Reply:
I believe it's all or one of these:
```
    .routePrograms()
    .routeAllProgramActions()
    .routeAllProgramFlows();
```

Reply:
Still can't get this to work.

I've tried several things, but just spinning my wheels. Latest attempt is here: https://github.com/RoundingWell/care-ops-frontend/pull/1249/commits/6f94a791fe0ecfa2e23d3983486ad0d2c6b65612.

In the `flow_app.js`, I can't get `this.flow.getProgramFlow()` to return the full program flow model. It just returns `{id: 'uuid'}` without any of the associated actions data.

Reply:
man..  I put you on a goose chase..  so.. I must've been looking at the add on patient dashboard.. this one is using that `programFlow.getAddableActions()` which is using the `includes`. 🤦   So the include was correct..........

Reply:
Okay I was wondering about that, but couldn't tell for sure. I reverted that last commit. I'll also need to fix some merge conflicts before this is ready as well.

Reply:
It would be more ideal to reset the head prior to the commit rather than reverting it for a new PR to reduce the blame depth.

Reply:
👍 , just reset it back.